Dockerfile

```diff
@@ -0,0 +1,12 @@
+# ----------------------------------
+# Pterodactyl Panel Dockerfile
+# ----------------------------------
+
+FROM golang:1.14-alpine
+COPY . /go/wings/
+WORKDIR /go/wings/
+RUN go build
```

Comment:
I've been using the following two commands to build the smallest binary possible, should reduce the image size by like 20 megs.

```
go build -ldflags="-s -w"
upx --brute wings
```

Reply:
Done